packages/components/src/internal/components/entities/actions.ts

```diff
@@ -943,9 +944,18 @@ export const getOriginalParentsFromLineage = async (
         Object.values(originalParents).forEach((parentTypes: List<EntityChoice>) => {
             originalParentTypeLsids.push(...parentTypes.map(parentType => parentType.type.lsid).toArray());
         });
+
+        // filter out additional parent alias columns that's already added
+        const additionalParentTypeSchemaQueryKeys = [];
+        additionalParentTypes?.forEach(parentType => {
+            additionalParentTypeSchemaQueryKeys.push(parentType.schemaName.toLowerCase() + '-' + parentType.queryName.toLowerCase())
```

Comment:
was there a reason for using the "-" separator here and at line 956 below? If these are SchemaQuery objects, you could likely use the toString() on them (which uses the | separator)